yt_idefix/data_structures.py

```diff
@@ -598,7 +598,6 @@ def _parse_definitions_header(self) -> None:
                 expr = re.sub(r"UNIT_(\w+)", self._get_unit, expr)
                 expr = re.sub(r"sqrt", "np.sqrt", expr)
                 expr = re.sub(r"log", "np.log", expr)
-                expr = re.sub(r"log10", "np.log10", expr)
```

Comment:
maybe the issue is that we're replacing "log", *then* "log10" ? in which case the comment on line 596 should be updated too ?

Reply:
to be clear, I think inverting the order of operations should do the trick

Reply:
As @dutta-alankar said [here](https://github.com/neutrinoceros/yt_idefix/pull/204#discussion_r1117888728), actually `re.sub(r"log", "np.log", expr)` is able to deal with both "log" and "log10", and even "log2". Because log10 contains the string "log", right?

Reply:
Oh that's right, good call